polling_stations/apps/data_collection/ems_importers.py

```diff
@@ -31,6 +32,10 @@
 class BaseXpressCsvImporter(BaseCsvStationsCsvAddressesImporter, metaclass=abc.ABCMeta):
     csv_delimiter = ","
 
+    # Set this to false in an import script if we want to only set a station
+    # point based on UPRN or co-ordinates (even if we've got a valid postcode)
+    allow_station_point_from_postcode = True
```

Comment:
I struggled to find a good name for this property as the behaviour it toggles is quite specific - suggestions for better name welcome if you've got any